app/main/routes.py

```diff
@@ -29,8 +34,35 @@ def download():
     if request.method == "GET":
         return render_template("download.html")
     if request.method == "POST":
-        data = get_remote_data(Config.DATA_STORE_API_HOST, "")  # specify endpoint here
-        return data
+        file_format = request.args.get("file_format")
+
+        if file_format not in ["json", "xlsx"]:
+            current_app.logger.error(
+                f"Unexpected file format requested from /download: {file_format}"
+            )
+            return abort(500), "Unknown file format."
+
+        response = get_response(
+            Config.DATA_STORE_API_HOST,
+            "/download",
+            query_params={"file_format": file_format},
+        )
+
+        if file_format == "json":
+            file_content = io.BytesIO(json.dumps(response.text).encode("utf-8"))
+            mimetype = "application/json"
```

Comment:
I think we might be able to infer the mimetype from the API response headers, rather than having to hardcode it here?

Reply:
ah nice idea